lib/salestation/app/errors.rb

```diff
@@ -7,41 +7,49 @@ class InvalidInput < Dry::Struct
         attribute :errors, Types::Strict::Hash
         attribute :hints, Types::Coercible::Hash.default({}.freeze)
         attribute? :debug_message, Types::Strict::String
+        attribute? :custom, Types::Coercible::Hash
```

Comment:
Maybe be should be `custom_data` instead, let me know which one sounds better.

Reply:
maybe even `details` as this provides more information about the error?

Reply:
so `details` will contain `error_details?`

Reply:
hmm. can't really think of a better name indeed.

Reply:
Although, to be fair salestation should not care what exactly this data contains. The main thing is that `custom/details` is merged with the rest of the attributes and not a standalone field.